.github/workflows/notify-mattermost-cloud-dependapot-pr.yml

```diff
@@ -0,0 +1,65 @@
+name: Notify Mattermost Cloud Team
+
+on:
+  workflow_call:
+    inputs:
+      commit:
+        description: "The commit used by the github checkout action. Default: github.sha"
+        type: string
+        default: ${{ github.sha }}
+      exit-with-status:
+        description: "Exit this job/workflow with the monitored job status. Options: true or false. Default: true"
+        type: string
+        default: "true"
+      highlight:
+        description: "Mattermost highlight. Default: pdcloudtestalerts"
+        type: string
+        default: "pdcloudtestalerts"
+      status:
+        description: "The monitored job, job status."
+        type: string
+        required: true
+    # Dependabot don't have this secrets and on PR's this secrets are not needed.
+    secrets:
+      MATTERMOST_WEBHOOK_URL:
+        required: false
+
+jobs:
+  check-old-dependabot-prs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Fetch old Dependabot PRs
+        id: fetch_prs
+        env:
+          GITHUB_TOKEN: ${{ secrets.GREENBONE_BOT_TOKEN }}
+        run: |
+          SEVEN_DAYS_AGO=$(date -d '7 days ago' +%s)
+          gh pr list --label "dependencies" --state "open" --json url,createdAt,title > all_dependabot_prs.json
+          
+          OLD_PRS=$(cat all_dependabot_prs.json | jq --argjson cutoff "$SEVEN_DAYS_AGO" \
+            '.[] | select((.createdAt | fromdateiso8601) < $cutoff)')
+
+          if [[ -z "$OLD_PRS" ]]; then
+            echo "OLD_PRS_FOUND=false" >> $GITHUB_ENV
+          else
+            echo "$OLD_PRS" | jq -r '. | "- **\(.title)**: [View PR](\(.url))"' > old_pr_list.txt
+            echo "OLD_PRS_FOUND=true" >> $GITHUB_ENV
+            echo "MESSAGE_TEXT=$(<old_pr_list.txt)" >> $GITHUB_ENV
+
+      - name: Notify Mattermost
+        if: env.OLD_PRS_FOUND == 'true'
+        uses: greenbone/actions/mattermost-notify@v3
+        with:
+          url: ${{ secrets.MATTERMOST_WEBHOOK_URL }}
+          channel: "pdcloudtestalerts"
+          message: ${{ env.MESSAGE_TEXT }}
+          repository: ${{ github.repository }}
+          commit: ${{ github.sha }}
+          branch: ${{ github.ref_name }}
+          status: "success"
```

Comment:
Shouldn't you use something like "warning" or so, to indicate, that there is action required?

Reply:
I checked the explanation of the status, `success` or `failure` that I can set, If I set `warning` is this still working?

Reply:
Actually no, you would need to add warning in https://github.com/greenbone/mattermost-notify/blob/main/mattermost_notify/status.py#L11 ...

You could use ⚠️ `:warning:` as the symbol ...
Or we just ignore my suggestion ...

Reply:
I misunderstood, thnx for clarification. I updated it again

Reply:
No, that won't work at this, you would need to contribute to the mentioned repository, in order to make this work!

Reply:
got it